addons/armaos/functions/fnc_terminal_addToHistory.sqf

```diff
@@ -0,0 +1,11 @@
+params ["_computer", "_commandString"];
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalCommandHistory = _terminal get "AE3_terminalCommandHistory";
+
+_terminalCommandHistory append [_commandString];
+
+_terminal set ["AE3_terminalCommandHistory", _terminalCommandHistory];
+
+_computer setVariable ["AE3_terminal", _terminal];
```

Comment:
Should be unnecessary, because variables referencing hashmaps are only pointers to the actual hashmap object, not the hashmap itself.

Reply:
removed in https://github.com/y0014984/Advanced-Equipment/pull/90/commits/996717f1cedd397770d94ec876aedb11e65687ad